src/include/miopen/solver/mlir_util.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef CK_MLIR_UTIL_HPP_
```

Comment:
Please add our standard license header and
```suggestion
#ifndef GUARD_MLIR_UTIL_HPP_
```

Reply:
Addressed this together with `implicitgemm_util.hpp` in c480a5a